src/modules/previewpane/powerpreview/trace.cpp

```diff
@@ -44,7 +44,7 @@ void Trace::PowerPreviewSettingsUpdated(LPCWSTR SettingsName, bool oldState, boo
 {
     TraceLoggingWrite(
         g_hProvider,
-        "PowerPreview_TweakUISettings_SuccesfullyUpdatedSettings",
+        "PowerPreview_TweakUISettings_SuccessfullyUpdatedSettings",
```

Comment:
@AirYang919 Can you confirm that this wont brake any telemetry reports?

Reply:
Looks good to me. Thanks for checking!